src/core/probe/kernel/kretprobe.rs

```diff
@@ -43,19 +45,25 @@ impl ProbeBuilder for KretprobeBuilder {
             bail!("Kretprobe builder already initialized");
         }
 
+        register_filter_handler(
```

Comment:
in the case we go for the lazy registration, this has to register two handlers one for kretprobes and another for the kprobes.

Reply:
not needed, see above